native/go/Makefile

```diff
@@ -23,4 +23,4 @@ include ../../mk/spksrc.native-cc.mk
 go_native_compile:
 	@$(MSG) "Building Go for host system"
 	# GO111MODULE disable go.mod per https://github.com/golang/go/issues/26996
```

Comment:
as GO111MODULE=off is dropped, this comment may be dropped too.

Reply:
nope, it ain't removed but rather moved on that command line further to the left :)